### ISSUE TITLE
TPE: Replace reference to browser specific implementations of cookies with desired outcome/behavior.

### ISSUE THREAD

**Issue by jasonanovak** [w3c/dnt#90]
In the July 23, 2018 Editor's Draft of Tracking Preference Expression (DNT)- Purposes Extension Addendum, the Introduction says:
> These would deliver data similar to that delivered by [ITP]'s "partitioned cookies" now implemented in Webkit browsers, but only writeable by top-level sites, and only when Site-Specific consent had been given for the named third-party domain. 

ITP and partitioned cookies is a Safari specific implementation.  The implementation and partitioning evolved between ITP 1.0 (https://webkit.org/blog/7675/intelligent-tracking-prevention/) and ITP 1.1 (https://webkit.org/blog/8142/intelligent-tracking-prevention-1-1/) and may evolve in the future.  There's also a difference in terminology that highlights the risk of incorporating browser-specific behavior in this note — Safari refers to  partitioned cookies while other browsers refer to double-keyed cookies and there may be implementation or behavioral differences across them.

Rather than cite an evolving browser specific feature, discussing the desired outcome / data to be passed would make this document more robust in the long term.

**Comment by michael-oneill** [comment]
Agreed. I have removed the reference to partitioned cookies and ITP, and put more emphasis on the advantages to first-parties and third-parties over using standard cookies
